src/router/index.js

```diff
@@ -58,4 +62,12 @@ router.beforeEach((to, from, next) => {
     });
 });
 
+const ifNotAuthenticated = (to, from, next) => {
+    if (firebase.auth().meta.requiresAuth) {
```

Comment:
beveik garantuoju jog `firebase.auth().meta` neegzistuoja.

Gerai apgalvok sita logika ir bandyk is naujo.

- kaip gali patikrinti dabartini user'i? https://firebase.google.com/docs/reference/js/firebase.auth.Auth#currentuser
```
firebase.auth().currentUser
```

- Kada negalima pasiekti /login ir /createUser?
Kai yra prisijunges user'is

Tai galvok kaip sitai sujungti